pkg/util/validation/limits.go

```diff
@@ -549,6 +551,11 @@ func (o *Overrides) AcceptHASamples(userID string) bool {
 	return o.GetOverridesForUser(userID).AcceptHASamples
 }
 
+// AcceptMixedHASamples returns whether the distributor should track and accept mixed samples from HA replicas for this user.
```

Comment:
Is it clearer if we say `should track and accept samples from mixed HA replicas`?